src/layouts/CreateProjectLayout.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from "react";
+import Grid from "@material-ui/core/Grid";
+import styled from "styled-components";
+
+import ProgressBar from "../../src/components/general/ProgressBar.js";
+import ProjectPaneContents from "../../src/components/project/ProjectPaneContents.js";
+import Button from "../../src/components/general/Button";
+import TEMP_ORGANIZATION_DATA from "./../../tmp-data/organizations.json";
+import TEMP_CATEGORY_DATA from "./../../tmp-data/categories.json";
+
+class CreateProjectLayout extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      organizations: TEMP_ORGANIZATION_DATA.organizations,
+      categories: TEMP_CATEGORY_DATA.categories,
+      progressPercentage: 0,
+      organization_id: "",
+      project_name: "",
+      project_location: "",
+      active_pane: 0
+    };
+  }
+
+  _handleOrganizationChange = event => {
+    this.setState({ organization_id: event.target.value });
+  };
+
+  _handleProjectNameChange = event => {
+    this.setState({ project_name: event.target.value });
+  };
+
+  _handleProjectLocationChange = event => {
+    this.setState({ project_location: event.target.value });
+  };
+
+  _incrementPane = () => {
+    this.setState({ active_pane: this.state.active_pane + 1 });
```

Comment:
If the new state depends on the previous state (like it does here), it's best to call `setState` with a function.

```js
this.setState((state) => ({
  activePaneIndex: state.activePaneIndex + 1
}))
```

In short, this is useful if multiple `setState`s get called in rapid succession. You can read more about that [in React's docs](https://reactjs.org/docs/react-component.html#setstate).